src/main/scala/com/gu/fulfilmentLookup/SalesforceCaseService.scala

```diff
@@ -73,7 +54,31 @@ object SalesforceCaseService extends CaseService with Logging {
     }
   }
 
-  override def raiseCase(config: Config, lookupRequest: LookupRequest, lookupResponseBody: LookupResponseBody): String \/ Boolean = {
+}
+
+trait RaiseCase {
+  def open(config: Config, lookupRequest: LookupRequest, lookupResponseBody: LookupResponseBody): String \/ Boolean
```

Comment:
if you call it "apply" then you can use it with RaiseCase(...) rather than RaiseCase.open(...)

it's a bit magic though 🎉 , but it can be useful - your choice!